src/shady-css/node-factory.js

```diff
@@ -79,13 +79,32 @@ class NodeFactory {
   }
 
   /**
+   * Creates an Expression Terms node.
+   * @param {Array} terms A list of terms.
+   * @return {object} An Expression Terms node.
+   */
+  expressionTerms(terms) {
+    return { type: nodeType.expressionTerms, terms };
+  }
+
+  /**
    * Creates an Expression node.
-   * @param {string} text The full text content of the expression (e.g.,
-   * `url(img.jpg)`)
+   * @param {string} value The value of the expression (e.g., `block`)
    * @return {object} An Expression node.
    */
-  expression(text) {
-    return { type: nodeType.expression, text };
+  expression(value) {
+    return { type: nodeType.expression, value };
+  }
+
+  /**
+   * Creates a Function node.
+   * @param {string} name The name of the function.
+   * @param {object} terms The Expression Terms node that represents the
+   * function arguments.
+   * @return {object} A Function node.
+   */
+  function(name, terms) {
```

Comment:
Potentially a bit confusingly here. I guess the alternatives are:

```javascript
function: function function(...) { ... }
```

Yeah, I guess this is ok. Probably.